### DISCUSSION TITLE
Benchmarking DuckDB against Sqlite for simple queries

### DISCUSSION THREAD

**Discussion by llimllib** [duckdb/duckdb#9698] [General]
Just thought folks around here would be interested to see this post: https://www.lukas-barth.net/blog/sqlite-duckdb-benchmark/

It concludes that for simple queries on the author's data set, sqlite greatly outperforms duckdb on queries that hit an index, and vice versa without an index.

(I'm not the author)

**Comment by Tishj** [comment]
Thanks for sharing, as the author mentions though, this is kind of comparing apples to oranges. Might be an interesting read none the less though 👍
